### DISCUSSION TITLE
A Getting started link became outdated

### DISCUSSION THREAD

**Discussion by skaunov** [arkworks-rs/.github#4] [General]
In Getting starting the last point is "[Quick introduction to programming with finite fields, elliptic curves, and pairings](https://github.com/Pratyush/algebra-intro) in arkworks." Since `~0.4.0` changed the interface significantly it doesn't really reflect current entities. Should the introduction be updated or this point changed to something fresher?

**Comment by Pratyush** [comment]
Thanks for catching this! 

I think this can be changed to point to the respective crate `READMEs`; haappy to accept a PR for this

**Reply by skaunov** [reply]
https://github.com/arkworks-rs/.github/pull/5
